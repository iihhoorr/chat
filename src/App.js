import React from 'react';

import ChatForm from './components/ChatForm';

const App = () => {
  return <ChatForm />;
};

export default App;
