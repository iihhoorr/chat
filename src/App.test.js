import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

test('App loads with default state', () => {
  const component = renderer.create(<App />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
