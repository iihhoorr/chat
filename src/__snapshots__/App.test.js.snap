// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App loads with default state 1`] = `
<div
  id="chat-container"
>
  <div
    id="chat-title"
  >
    <img
      alt="Elixirator"
      className="post-avatar"
      src="https://media.istockphoto.com/vectors/no-image-available-sign-vector-id922962354?k=20&m=922962354&s=612x612&w=0&h=f-9tPXlFXtz9vg_-WonCXKCdBuPUevOBkp3DQ-i0xqo="
    />
    <span>
      Elixirator
    </span>
  </div>
  <div
    id="chat-message-list"
  >
    <div
      className="message-row you-message"
    >
      <div
        className="message-content"
      >
        <div
          className="message-text"
        >
          yep
        </div>
        <div
          className="message-time"
        >
          Dec 22
        </div>
      </div>
    </div>
    <div
      className="message-row other-message"
    >
      <div
        className="message-content"
      >
        <img
          alt="Elixirator"
          src="https://media.istockphoto.com/vectors/no-image-available-sign-vector-id922962354?k=20&m=922962354&s=612x612&w=0&h=f-9tPXlFXtz9vg_-WonCXKCdBuPUevOBkp3DQ-i0xqo="
          style={
            Object {
              "opacity": "1",
            }
          }
        />
        <div
          className="message-text"
        >
          again question...
        </div>
        <div
          className="message-time"
        >
          Dec 22
        </div>
      </div>
    </div>
    <div
      className="message-row you-message"
    >
      <div
        className="message-content"
      >
        <div
          className="message-text"
        >
          too busy?
        </div>
        <div
          className="message-time"
        >
          Dec 22
        </div>
      </div>
    </div>
    <div
      className="message-row you-message"
    >
      <div
        className="message-content"
      >
        <div
          className="message-text"
        >
          And?...)
        </div>
        <div
          className="message-time"
        >
          Dec 21
        </div>
      </div>
    </div>
    <div
      className="message-row other-message"
    >
      <div
        className="message-content"
      >
        <img
          alt="Elixirator"
          src="https://media.istockphoto.com/vectors/no-image-available-sign-vector-id922962354?k=20&m=922962354&s=612x612&w=0&h=f-9tPXlFXtz9vg_-WonCXKCdBuPUevOBkp3DQ-i0xqo="
          style={
            Object {
              "opacity": "0",
            }
          }
        />
        <div
          className="message-text"
        >
          Too many questions...
        </div>
        <div
          className="message-time"
        >
          Dec 21
        </div>
      </div>
    </div>
    <div
      className="message-row you-message"
    >
      <div
        className="message-content"
      >
        <div
          className="message-text"
        >
          What have you been up to lately?
        </div>
        <div
          className="message-time"
        >
          Dec 21
        </div>
      </div>
    </div>
    <div
      className="message-row you-message"
    >
      <div
        className="message-content"
      >
        <div
          className="message-text"
        >
          How’s everything?
        </div>
        <div
          className="message-time"
        >
          Dec 20
        </div>
      </div>
    </div>
    <div
      className="message-row you-message"
    >
      <div
        className="message-content"
      >
        <div
          className="message-text"
        >
          What’s new?
        </div>
        <div
          className="message-time"
        >
          Dec 20
        </div>
      </div>
    </div>
    <div
      className="message-row other-message"
    >
      <div
        className="message-content"
      >
        <img
          alt="Elixirator"
          src="https://media.istockphoto.com/vectors/no-image-available-sign-vector-id922962354?k=20&m=922962354&s=612x612&w=0&h=f-9tPXlFXtz9vg_-WonCXKCdBuPUevOBkp3DQ-i0xqo="
          style={
            Object {
              "opacity": "0",
            }
          }
        />
        <div
          className="message-text"
        >
          Hello
        </div>
        <div
          className="message-time"
        >
          Dec 20
        </div>
      </div>
    </div>
    <div
      className="message-row you-message"
    >
      <div
        className="message-content"
      >
        <div
          className="message-text"
        >
          Hi
        </div>
        <div
          className="message-time"
        >
          Dec 20
        </div>
      </div>
    </div>
  </div>
  <form
    id="chat-form"
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      placeholder="type a message"
      type="text"
      value=""
    />
    <button
      className="primary-button"
      disabled={true}
      type="submit"
    >
      Send
    </button>
  </form>
</div>
`;
